### PR TITLE
Adding support for openai_organization env variable

### PR DESCRIPTION
**Pull request by kaneda2004**
Addresses #270

### CODE REVIEW

File: env.template
Comment (Torantulino) [inline env.template:1]: This file is named incorrectly. 
It should be `.env.template`

**Review by Torantulino — CHANGES_REQUESTED** [review]

File: env.template
Comment (kaneda2004) [reply env.template:1]: Thank you - my apologies - amended.

**Comment by nponeccop** [comment]
@kaneda2004 Resolve the conflicts

**Review by nponeccop — APPROVED** [review]

**Comment by nponeccop** [comment]
@kaneda2004 Resolve the conflicts, it got outdated again due to other merges.

**Comment by anonhostpi** [comment]
I think this needs to be done after the re-arch. Switching APIs is a proposed feature, and I believe that adding/hotswitching APIs is a method done best after the team finishes implementing APIs as plugins rather than internal parts of AutoGPT.

The reason for this is that there are so many APIs to switch out, that there seems to be a need to loosen how tightly coupled AutoGPT is to each _individual_ API

**Comment by Boostrix** [comment]
As long as it's strictly config only, or introducing new options, should be pretty safe to consider even prior to the /during the re-arch

**Comment by p-i-** [comment]
This looks sensible.

**Review by richbeales — DISMISSED** [review]

**Review by merwanehamadi — APPROVED** [review]

**Pull request merged**